o4/version.py

```diff
@@ -3,13 +3,13 @@
 
 import datetime
 
-VERSION = (1, 2, 70)
-VERSION_STR = '1.2.70'
+VERSION = (1, 2, 138)
+VERSION_STR = '1.2.138'
```

Comment:
There's an increase. :)

Reply:
Yep, since every debugging change bumped it, too. Do you think I should just manually make it an increment compared to the previous release?